File: Linguagem C Exercicios/C-Sequencial/RETANGULO.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    double base,altura,area,perimetro,diagonal = 0;


    printf("Base do retangulo: ");
    scanf("%lf",&base);
    printf("Altura do retangulo: ");
    scanf("%lf",&altura);

    area = base * altura;
    perimetro = (base*2)+(altura*2);
    printf("Area do retangulo: %.4lf\n",area);
    printf("Perimetro do retangulo: %.4lf\n",perimetro);

    diagonal = sqrt((base*base) + (altura*altura));
    printf("Diagonal do retangulo: %.4lf",diagonal);


    return 0;
}

File: Linguagem C#/Sequencial/Retangulo.cs
using System;
using System.Globalization;

namespace Retangulo {
    class Program {
        static void Main(string[] args) {

            CultureInfo CI = CultureInfo.InvariantCulture;
            double Base, altura, area, perimetro, diagonal;

            Console.Write("Base do retangulo: ");
            Base = double.Parse(Console.ReadLine(),CI);
            Console.Write("Altura do retangulo: ");
            altura = double.Parse(Console.ReadLine(),CI);

            area = altura * Base;
            perimetro = 2 * (altura + Base);
            diagonal = Math.Sqrt(Math.Pow(altura,2) + Math.Pow(Base,2));

            Console.WriteLine("AREA = "+area.ToString("F4",CI));
            Console.WriteLine("PERIMETRO = "+perimetro.ToString("F4",CI));
            Console.WriteLine("DIAGONAL = "+diagonal.ToString("F4",CI));
        }
    }
}
